libats-slick/src/test/scala/com/advancedtelematic/libats/slick/db/SlickExtensionsSpec.scala

```diff
@@ -104,7 +104,7 @@ class SlickExtensionsSpec extends FunSuite with Matchers with ScalaFutures with
   test("handleForeignKeyError") {
     val error = new RuntimeException("[test] expected error") with NoStackTrace
     val g = BookMeta(1, 1984, 0)
-    val f = db.run((bookMeta += g).handleForeignKeyError(error))
+    val f = db.run((bookMeta += g).mapError { case SqlExceptions.NoReferencedRow(_) => error })
```

Comment:
would still be nice to have a shorthand for `handleForeignKeyError`, which could be written in terms of `mapError`